podvm/files/etc/aa-offline_fs_kbc-keys.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What's the reason to have this file by default ?

Reply:
The CDH currently has the explicit assumption that this file exists and tries to read it, so without having a dummy value , we hit the failure:
```
# /usr/local/bin/confidential-data-hub
Error: init Hub failed: kbs client creation failed: Kbs client error: offline-fs-kbc: read /etc/aa-offline_fs_kbc-keys.json failed: No such file or directory (os error 2)
```

(more info in the issue discussion https://github.com/confidential-containers/cloud-api-adaptor/issues/1424#issuecomment-1729193022)

It's worth noting that kata-containers with local hypervisor also hits this, so we might end up having to change the CDH behaviour to handling that missing file, but rather than we blocked on it I just provided a workaround here.

Reply:
Just for my understanding, is this error hit only when using offline_fs_kbc or also with cc_kbc ?
And will it create problem if using cc_kbc ?
I'm currently trying with cc_kbc, so will know. But just in case you are already familiar

Reply:
Yes, I believe that's the case, Ding has suggested in DMs that it's not needed when using cc_kbc

Reply:
Can we have issue created to clean this up for later?